README.md

```diff
@@ -0,0 +1,94 @@
+# TCTrack
+
+The TCTrack package provides the functionality for running tropical cyclone (TC)
+tracking algorithms and analysing the results. This can be used to compare the output
+of different algorithms for a variety of data sources.
```

Comment:
```suggestion
The TCTrack package aime to provide functionality for running tropical cyclone (TC)
tracking algorithms and analysing the results. This can be used to compare the output
of different algorithms for a variety of data sources.
```
Consider softening this statement for accuracy now?